lib/tasks/gettext_task_override.rb

```diff
@@ -1,13 +1,47 @@
 module GettextI18nRailsJs
   module Task
+    # use combined .po
     def files_list
       require "gettext_i18n_rails/tasks"
 
       ::Pathname.glob(::File.join(locale_path, 'combined', "**", "*.po"))
     end
 
+    # output to ui-classic, not Rails.root; oldjs, not assets/javascripts
     def output_path
-      ::ManageIQ::UI::Classic::Engine.root.join(GettextI18nRailsJs.config.output_path)
+      ::ManageIQ::UI::Classic::Engine.root.join('app/javascript/oldjs/locale')
+    end
+
+    # use path/lang.json instead of path/lang/app.js
+    def destination(lang)
+      output_path.mkpath
+
+      path = output_path.join("#{lang}.json")
+      path.open("w") do |f|
+        f.rewind
```

Comment:
This is not necessary.  Files opened for write should always be at the start of the file.

Reply:
Good to know, removing :) This was copied from the method we're overriding.